.github.off/workflows/release-tags.yml

```diff
@@ -0,0 +1,14 @@
+
+name: Release tagged build
+
+on:
+  push:
+    tags: [ '*' ]
+
+permissions:
+  contents: write
+
+jobs:
+  release-tags:
+    uses: GTNewHorizons/GTNH-Actions-Workflows/.github/workflows/release-tags.yml@master
```

Comment:
What does this do exactly?

Reply:
Remember that this is in the .github.off directory so it's not used, this entire directory can be deleted if you don't plan to use GitHub actions, but if you do use GitHub actions then what this specifically does is run that ci script which completely sets up a Java build environment, compiles the mod in full, runs its tests if any, and runs its publish scripts to publish to all of the various places, but if you're using your own CI then you'll do it however you would on your CI.  This is just left here as an example and documentation, or if you ever plan to use GitHub actions in the future.